org-brain.el

```diff
@@ -264,18 +269,25 @@ Ignores \"dotfiles\"."
 (defun org-brain-headline-entries ()
   "Get all org-brain headline entries."
   (unless org-id-locations (org-id-locations-load))
-  (let (ids)
-    (maphash
-     (lambda (id file)
-       (when (and (string-prefix-p (expand-file-name org-brain-path)
-                                   (expand-file-name file))
-                  (not (org-brain-id-exclude-taggedp id)))
-         (let* ((heading (org-id-find id t))
-                (name (org-entry-get heading "ITEM")))
-           (push (list (org-brain-path-entry-name file) name id)
-                 ids))))
-     org-id-locations)
-    ids))
+  (save-window-excursion
+	(let (ids)
+	  (dolist (file (org-brain-files) ids)
+		(find-file file)
```

Comment:
I would probably use `(with-current-buffer (find-file-noselect file)...` instead of `save-window-excursion`. Not sure if it is better or worse though.

Reply:
Thanks! Changed. I haven't found a perceptible performance impact by making this change, but this expresses the intent a little better.